packages/calcite-components/src/components/action-menu/action-menu.scss

```diff
@@ -37,4 +47,8 @@
   max-h-menu;
 }
 
+calcite-popover {
+  // TODO: add sub-component tokens
```

Comment:
Should these be added as part of this PR? The popover vars already exist.

Reply:
Popover still needs to be refactored. It's next on my list. I was going to take care of it then.